app/src/main/kotlin/compiler/semantic_analysis/VariablePropertiesAnalyzer.kt

```diff
@@ -77,8 +85,12 @@ object VariablePropertiesAnalyzer {
                 is Function -> {
                     node.parameters.forEach {
                         variableProperties[it] = VariableProperties(node)
```

Comment:
@ignacy123 are you sure that this should be (`node`) instead of (`currentFunction`)? Default variables are computed by the parent functions as far as I understand.

Reply:
Ok